builder/exec.go

```diff
@@ -4,8 +4,12 @@
 package builder
 
 import (
+	"fmt"
+	"log"
 	"os"
 	"os/exec"
+
+	"github.com/morikuni/aec"
```

Comment:
What OS/Terminals have you validated this on? Mainly have you checked what it looks like on Windows Cmd and Powershell?

Reply:
I added `aec` for version/new commands which rendered on Windows cmd + PowerShell on Windows 10. Would be good to have it checked on one of your VMs if you have time.

Reply:
I’ve only tested in macOS in iTerm and Terminal. I don’t have any Linux vm’s on hand but can spin one up if necessary.